hover-engine.js

```diff
@@ -87,8 +87,8 @@ class HoverEngine {
     return this
   }
 
-  notifyListeners() {
-    this.listeners.forEach(listener => listener(this.store, this.actions))
+  notifyListeners(actionName, actionArguments) {
```

Comment:
## Changes
`notifyListeners` now takes in
* `actionName` the name of the function called (as a string)
* `actionArguments` the argument passed into the function call